packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,524 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
++ import { textXSmallStyle, textSmallStyle, textMediumStyle, textLargeStyle, textXLargeStyle } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+**Colors**
+
+As of v3 of the Porsche Design System the color theme got reworked completely to achieve a monochrome look. To get an
+overview of the new colors have a look at [Theme](https://designsystem.porsche.com/latest/styles/theme).
+
+- `lightTheme` and `darkTheme` got renamed to `themeLight` and `themeDark`. In addition, you can also single import a
+  theme if for e.g. you only need `lightTheme`.
+
+```diff
+- import { color } from '@porsche-design-system/utilities';
++ import { theme, themeLight, themeDark } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+As the design tokens for `themeLight` and `themeDark` are in sync, the name changes mentioned below concern both themes
+even if only light theme is displayed in the diff.
+
+- `brand` got renamed to `primary`.
+
+```diff
+- color.lightTheme.brand
++ themeLight.primary
+```
+
+- `default` is removed.
+
+```diff
+- color.lightTheme.default
+```
+
+- `backgound.default` is renamed to `background.base`.
+
+```diff
+- color.lightTheme.background.default
++ themeLight.background.base
+```
+
+- `neutralContrast` is renamed to `contrast`.
+
+```diff
+- color.lightTheme.neutralContrast.{low|medium|high}
++ themeLight.contrast.{low|medium|high}
+```
+
+- `notification.neutral` and `notification.neutralSoft` are renamed to `notification.info` and `notification.infoSoft`.
+
+```diff
+- color.lightTheme.notification.neutral
+- color.lightTheme.notification.neutralSoft
+- themeLight.notification.info
+- themeLight.notification.infoSoft
+```
+
+- `state.focus` is changed from `currentColor` to a fixed color, depending on the theme.
+
+**Font**
+
+- `fontFamily` is provided as design token
+
+```diff
+- import { fontFamily } from '@porsche-design-system/utilities';
++ import { fontFamily } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+- `fontWeight` is provided as design token
+
+```diff
+- import { fontWeight } from '@porsche-design-system/utilities';
++ import { fontWeight } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+**Spacings**
+
+With the upcoming v3 release, we will be introducing fluid spacers alongside our static spacers. These spacers will
+adjust in size based on a min and max value, providing a linear response to changes in browser width. Additionally,
+we've switched from using _rem_ to _px_ for our spacers to ensure they don't unnecessarily expand when the root font
+size is adjusted.
+
+To get an overview ot the new spacings have a look at [Spacing](https://designsystem.porsche.com/latest/styles/spacing).
+
+```diff
+- import { spacing } from '@porsche-design-system/utilities';
+- import { layout } from '@porsche-design-system/utilities';
```

Comment:
Here we have trailing semi colons `;` in other snippets we don't. Let's stay consistent.